spec/lib/upgrader_spec.rb

```diff
@@ -16,9 +16,11 @@
   end
 
   describe 'latest_version_raw' do
-    it 'should be latest version for GitLab 5' do
+    it 'should be latest version for GitLab' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.